### PR TITLE
[READY] Improvised Makeshift Pistol

### PR DESCRIPTION
**Pull request by ghost**
NOTE: This is a re submission of a previous PR. The reasoning behind the resubmitting is due to the TG version of this PR being closed and not merged and the original PR being outdated. Due to GitHub's problems with forking I was not able to change the original PR so I submitted this one.

## About The Pull Request

This pull requests adds a new firearm in the default crafting menu. This pistol is chambered in standard 9mm bullets and can hold up to 5 bullets in its magazine. The pistol fires at a delay of 1 second and uses the standard bolt you can find on the Makarov pistol. To craft it, you need 2 Plasteel sheets, 5 Metal sheets, 1 Modular Receiver and a pipe. It comes with a magazine but you can craft more using 2 Plasteel Sheets and 2 Metal Sheets for a magazine. Whether or not you will be able to craft magazines in the future is to be seen as this may cause a balance issue.

In the future the recipe will be changed as I will add mechanics such as a hot gun which burns you if you dont have appropriate gloves or misfiring/exploding.

This has been tested and the pistol is working.

## Why It's Good For The Game

Previous to this PR, there were 2 ranged firearms that you could craft by default. The Flamethrower and the Makeshift Shotgun. While both of these weapons fill their niches perfectly (Flamethrower for crowd control/short range and shotgun for short range), there are no weapons to be used for medium to long range combat. While yes, you can outfit the shotgun with a slug and use it for medium/long range, the fact that you only have a single shot before you need to reload puts you at a massive disadvantage since if you miss you will need to manually reload before being able to shoot again. With the pistol, if you miss you can simply lead and shoot again for another 4 times before needing a reload. You are still at a disadvantage to people using proper weapons however since they will most likely have higher capacity weapons with higher bullet velocities than the pistol's.

This weapon should be treated as CONTRABAND by any Security Officer that sees someone holding it, since it can do quite a bit of damage if used in the right hands. Arrest is warranted if seen with one.

TL;DR
This is good for the game because it fills a niche the other craftable weapons cannot.

Below are some images of the pistol.

image

Note that the below image is an earlier sprite of the gun. The off-centered sprite has also been fixed.

image

## Changelog
:cl:
add: New gun sprite
add: More difficult recipe
/:cl:

### CODE REVIEW

**Comment by BarrelOfNapkins** [comment]
I am just assuming this does 20 damage/shot?
If so, I think 5 rounds is too much, as that can put someone into crit. When looking at other makeshift ranged weapons, like the shotgun, even it cannot put someone into crit without being reloaded.

**Comment by ghost** [comment]
> I am just assuming this does 20 damage/shot?
> If so, I think 5 rounds is too much, as that can put someone into crit. When looking at other makeshift ranged weapons, like the shotgun, even it cannot put someone into crit without being reloaded.

If youre quick enough with the shotgun, you can easily down someone in two shots since its 75 damage with buckshot. The reload for it is as fast as you can click on it with buckshot. You can also duel wield and do an astounding 150 brute. The pistol however has a delay of one second and its magazine has to be MANUALLY loaded one bullet at a time (multi_load was set to false)

**Comment by BarrelOfNapkins** [comment]
> > I am just assuming this does 20 damage/shot?
> > If so, I think 5 rounds is too much, as that can put someone into crit. When looking at other makeshift ranged weapons, like the shotgun, even it cannot put someone into crit without being reloaded.
> 
> If youre quick enough with the shotgun, you can easily down someone in two shots since its 75 damage with buckshot. The reload for it is as fast as you can click on it with buckshot. You can also duel wield and do an astounding 150 brute. The pistol however has a delay of one second and its magazine has to be MANUALLY loaded one bullet at a time (multi_load was set to false)

You are talking about if the person is point blank range? Then yes.
But the advantage of a ranged weapon is not point blank range.
This seems like the strongest makeshift weapon introduced into the game. If you intend to nerf it, then nerf it in this PR, otherwise I do not understand why you intend to introduce something to the game that you know is too strong.

**Comment by ghost** [comment]
@tanory07 The intention was, as described in the PR, to fill the medium/long range role that the others cannot fill. The ability to shoot more than once is to compensate for the fact that at those ranges there's a high chance you can miss. At closer ranges, this weapon is less powerful than the shotgun.

**Comment by BarrelOfNapkins** [comment]
I agree, but compare this pistol to something that is a next tier up, the detective's colt. Then compare the makeshift shotgun to the next tier up, the double barrel.
Colt advantage is 20% more ammo and can fire faster
Double barrel advantage is double the ammo #
Consider nerfing the ammo count until you nerf the other parts of this weapon

**Comment by ghost** [comment]
@remove32 Merge?
